config/environments/development.rb

```diff
@@ -27,7 +27,7 @@
   end
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
I think we should probably set this to "false" in `development`, since we don't want email to be sent.